chirp/drivers/ft450d.py

```diff
@@ -451,13 +494,14 @@ def _clone_in(self):
             else:
                 repeat = 1
             for _i in range(0, repeat):
-                data += self._read(block, blocks)
-                self.pipe.write(chr(CMD_ACK))
+                chunk = self._read(block, blocks)    # returns bytes()
+                data += chunk
+                self.pipe.write(bytes(chr(CMD_ACK), encoding='utf8'))
                 blocks += 1
                 status.cur = blocks
                 self.status_fn(status)
-        data += self.MODEL      # Append ID
-        return memmap.MemoryMap(data)
+        data += bytes(self.MODEL, encoding='utf8')      # Append ID

```

Comment:
It would be more conventional to use `self.MODEL.encode()` here. However, you do *not* want a UTF-8 string, you want a direct conversion, so `self.MODEL.enocde('ascii') would be better.